apps/ex_wire/lib/ex_wire/rank/peer_details.ex

```diff
@@ -0,0 +1,17 @@
+defmodule ExWire.Rank.PeerDetails do
+  @moduledoc """
+  A data structure that will help us establish a peers value.
+  """
+  alias ExWire.Struct.Peer
+
+  @type t :: %__MODULE__{
+          peer: Peer.t() | nil,
+          connection_duration: Time.t() | nil,
+          sent_message_count: non_neg_integer(),
+          received_message_count: non_neg_integer()
+        }
+  defstruct peer: nil,
+            connection_duration: nil,
+            sent_message_count: 0,
+            received_message_count: 0
```

Comment:
this two fields are currently not used, right? `sent_message_count` and `received_message_count`

Reply:
No. That’s what I’m aiming next!